incubator/kafka/templates/tests/test_topic_create_consume_produce.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ .Release.Name }}-test-topic-create-consume-produce"
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+  - name: {{ .Release.Name }}-test-consume
+    image: {{ .Values.image }}:{{ .Values.imageTag }}
+    command:
+    - sh
+    - -c
+    - |
+      # Create the topic
+      kafka-topics --zookeeper {{ template "zookeeper.url" . }} --topic helm-test-topic-create-consume-produce --create --partitions 1 --replication-factor 1 --if-not-exists && \
+      # Create a message
+      MESSAGE="`date -u`" && \
+      # Produce a test message to the topic
+      echo $MESSAGE | kafka-console-producer --broker-list {{ template "kafka.fullname" . }}:9092 --topic helm-test-topic-create-consume-produce && \
+      # Produce a test message to the topic
+      kafka-console-producer --broker-list {{ template "kafka.fullname" . }}-headless:9092 --topic helm-test-topic-create-consume-produce && \
```

Comment:
Rm lines 21-22

Reply:
@benjigoldberg are you wanting to do this before this gets merged?

Reply:
@viglesiasce yeah. I pushed this up and then noticed this on my commute. I was tinkering with the tests and forget to delete these two lines